### DISCUSSION TITLE
Is there any way to put and then access this cheatsheet on a computer without internet access?

### DISCUSSION THREAD

**Discussion by jpozderac** [wilfredinni/python-cheatsheet#232] [Q&A] [answered]
Hello Community!

I know I can clone or download the repository that contains all of the information in python-cheatsheet. 

However, is there a way to then view it, with all of the nice formatting and search-ability, on a computer that doesn't have access to the internet?

Thanks,
Jon

**Comment by wilfredinni** [comment, marked as answer]
Hi! Yes, you can. The website is actually a PWA, which means you can install it on your pc/android device:

![image](https://user-images.githubusercontent.com/23016174/231483241-f8d49e6c-3ab8-4ebb-8ddd-7682bcc2f9b0.png)

Python cheatsheet will be available offline and update automatically on new changes.

**Reply by jpozderac** [reply]
Nice! Thank you for the response!

Follow-up:
Can I just "install" it and then put it on a flash drive to get it to the non-networked computer? One hiccup i see already is that if I "install" it on my Mac, it may not work on the Windows machine that is non-networked (sorry these details weren't included in the first post).

**Comment by jpozderac** [comment]
I am not sure if should start a new thread with my follow-up:

Follow-up:
Can I just "install" it and then put it on a flash drive to get it to the non-networked computer? One hiccup i see already is that if I "install" it on my Mac, it may not work on the Windows machine that is non-networked (sorry these details weren't included in the first post).

**Reply by wilfredinni** [reply]
Hi. I am afraid you cannot do that. The alternative would be for you to convert the sections you are interested in to PDF and save them. Later this year, I will be adding the ability to download the complete thing to EPUB/PDF.

**Reply by jpozderac** [reply]
Roger that. Thanks again for following up!
